docs/docsite/known_good_reqs.txt

```diff
@@ -4,7 +4,7 @@
 #    test/sanity/code-smell/docs-build.requirements.txt
 #    test/sanity/code-smell/rstcheck.requirements.txt
 
-antsibull == 0.41.0
+antsibull == 0.42.0b1
```

Comment:
Why a beta release? Is this just for testing in this PR before merging with a stable version?

Reply:
Because it contains a minimal fix needed to unblock #74963. A better fix (resp. a proper implementation for ansible-core 2.13+ which stops using internal APIs) requires #77035 to be merged, but I was told this will not happen before #74963 is merged.